skbase/base/_base.py

```diff
@@ -851,3 +856,101 @@ def check_is_fitted(self):
                 f"This instance of {self.__class__.__name__} has not been fitted yet. "
                 f"Please call `fit` first."
             )
+
+    def get_fitted_params(self):
+        """Get fitted parameters.
+
+        State required:
+            Requires state to be "fitted".
+
+        Returns
+        -------
+        fitted_params : dict of fitted parameters, keys are str names of parameters
+            parameters of components are indexed as [componentname]__[paramname]
+        """
+        if not self.is_fitted:
+            raise NotFittedError(
+                f"estimator of type {type(self).__name__} has not been "
+                "fitted yet, please call fit on data before get_fitted_params"
+            )
+
+        fitted_params = {}
+
+        def sh(x):
+            """Shorthand to remove all underscores at end of a string."""
+            if x.endswith("_"):
+                return sh(x[:-1])
+            else:
+                return x
+
+        # add all nested parameters from components that are sktime BaseObject
```

Comment:
Should be objects that are skbase BaseEstimator's

Reply:
fixed